### ISSUE TITLE
Fedora packaging test failure for 1.3.0 release

### ISSUE THREAD

**Issue by tasleson** [libstorage/libstoragemgmt#149]
While trying to package 1.3.0 for fedora I'm running into the following error (fedpkg mockbuild):

```
96%: Checks: 50, Failures: 2, Errors: 0
tester.c:3834:F:Basic:test_local_disk_ident_led:0: lsm_local_disk_ident_led_on(): Got unexpected return: 209
tester.c:3841:F:Basic:test_local_disk_fault_led:0: lsm_local_disk_fault_led_on(): Got unexpected return: 209
Fail exit[1]: /tmp/28573/bin//tester
```

`209 == LSM_ERR_NOT_FOUND_DISK`

**Comment by tasleson** [comment]
When running in the context of the fakeroot I'm seeing:

```
>>> import lsm
>>> lsm.LocalDisk.list()
['/dev/sda']
>>> lsm.LocalDisk.fault_led_on('/dev/sda')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/builddir/build/BUILD/libstoragemgmt-1.3.0/python_binding/lsm/_local_disk.py", line 306, in fault_led_on
    return _use_c_lib_function(_local_disk_fault_led_on, disk_path)
  File "/builddir/build/BUILD/libstoragemgmt-1.3.0/python_binding/lsm/_local_disk.py", line 28, in _use_c_lib_function
    raise LsmError(err_no, err_msg)
lsm._common.LsmError: NOT_FOUND_DISK(209): Disk /dev/sda not found
```

**Comment by tasleson** [comment]
@cathay4t Gris, please take a look at this.  I think this difference in error could be due to running as root vs. running as non-root.  However, does it really make sense to try and test these led functions on virtual or physical hardware that doesn't support it?

**Comment by cathay4t** [comment]
I will skip led when run by non-root.
And this is a bug as '/dev/sda' do exist.

**Comment by cathay4t** [comment]
The fedora/RHEL mock chroot has '/sys' mounted while '/dev' is not.

Patch is on the way.

**Comment by cathay4t** [comment]
Fixed in PR #150, closing this issue.
